### DISCUSSION TITLE
Separate issues on the "Object has been destroyed" error

### DISCUSSION THREAD

**Discussion by stankiewiczmichal** [FreeTubeApp/FreeTube#6471] [General]
From what I understand, the “Object has been destroyed” error is caused by many different bugs, and it's just such a universal alert. That being said, I think that keeping all the reports on this topic in one issue #4762 could be problematic, both for developers and reporters.
Perhaps it would be useful to separate them and divide them into the reasons why this bug occurs?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
We would like to keep it there so its easier for users to find.
